rust_src/src/symbols.rs

```diff
@@ -6,9 +6,10 @@ use std::fmt::{Debug, Formatter};
 use remacs_macros::lisp_fn;
 
 use crate::{
-    buffers::LispBufferLocalValueRef,
+    buffers::per_buffer_idx_from_field_offset,
```

Comment:
these are unused right ?

Reply:
yeah, I pull this PR off of your branch after hacking on it locally.